docs/installation.md

```diff
@@ -1,31 +1,49 @@
 ## Installation
 
 ### Where should I install wp-browser?
-As a rule-of-thumb wp-browser should be installed in the root folder of your project.  
+
+As a rule-of-thumb wp-browser should be installed in the root folder of your 
```

Comment:
Hard-wrap here and not elsewhere? On purpose? Why?

Reply:
This one is mainly for clearly identify the next section but does not have any impact on the output.